lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -57,6 +57,8 @@ function getHTMLImages(allElements) {
 
   return allImageElements.map(element => {
     const computedStyle = window.getComputedStyle(element);
+    const isPicture = !!element.parentElement && element.parentElement.tagName === 'PICTURE';
```

Comment:
```suggestion
    const isPicture = element.parentElement && element.parentElement.tagName === 'PICTURE';
```

ouch ur hurting me

Reply:
the type needs to be a boolean when it's used below

do you prefer?
```suggestion
    const isPicture = Boolean(element.parentElement && element.parentElement.tagName === 'PICTURE');
```

Reply:
this was also just a straight up moved line btw 😛